apps/docs/package.json

```
@@ -5,7 +5,7 @@
   "description": "",
   "scripts": {
     "dev": "nodemon --config nodemon.config.json -x 'typedoc && vitepress dev'",
-    "build": "typedoc && vitepress build",
+    "build": "typedoc && vitepress build && tsx ./scripts/purge-docs.ts",
```

The purge needs to go before the vite build to avoid creating HTML files for deleted items.
```suggestion
    "build": "typedoc && tsx ./scripts/typedoc-postbuild.ts && vitepress build",
```